src/App.vue

```diff
@@ -166,11 +311,39 @@ export default {
         tables: [],
         tableFilter: '',
         rules: {
-            password: [{
-                required: true,
-                message: 'Please input password',
-                trigger: 'blur'
-            }]
+            password: [
+                { required: true, message: 'Please input password', trigger: 'blur'}
+            ],
+        },
+        ruleName: {
+            name: [
+                { required: true, message: 'Please input the name', trigger: 'blur'}
+            ],
+        },
+        ruleRow: {
+            name: [
+                { required: true, message: 'Please input the column name', trigger: 'blur'}
+            ],
+            type: [
+                { required: true, message: 'Please input the column type', trigger: 'blur'}
+            ],
+        },
+        ruleTable: {
+            select: [
+                { required: true, message: 'Please select', trigger: 'change'}
+            ],
+        },
+        newName: {
+            name: '',
+        },
+        inputColumn: [
+            {id: '', name: '', type: ''},
+        ],
+        tableSelected: {
+            select: '',
+        },
+        databaseSelected: {
+            select: '',
```

Comment:
On these cases, since it's only one line, I believe it's better to leave all on the same line. But don't need to change, just a head's up

```js
key: { key: 'value' },
```